src/test.cpp

```diff
@@ -1,63 +0,0 @@
-#include "engine/engine.h"
```

Comment:
I think this script is quite useful when we want to change the arguments such as config files and threadNum, and see the outcomes. Is there any substitute (maybe a python script)?

Reply:
I revert the cpp file in `tool/debug` since it is more likely a debug tool rather than a test. And it is available when `CMAKE_BUILD_TYPE` is `Debug`